js-src/package.json

```diff
@@ -31,7 +31,6 @@
     "core-js": "^2.4.1",
     "reflect-metadata": "^0.1.8",
     "rxjs": "5.0.0-beta.12",
-    "scriptjs": "^2.5.8",
```

Comment:
this seems wrong, we still need script js for the channel API, i have no idea why this was deleted